### DISCUSSION TITLE
Two filters with the same class are not syncing values between eachother

### DISCUSSION THREAD

**Discussion by KasparRosin** [laravel/nova-issues#4508] [Ideas & Feature Requests]
- Nova Version: 4.9.2

### Description:
Two of the same filters (with the same filter-key aka class) do not sync values between each other.

https://user-images.githubusercontent.com/33309407/176368493-351ee24e-4df8-4d7e-97ae-7824085cf0e4.mp4
In laravel nova version 3, you can see that the two identical filters sort-of sync values between each other. 
Though the example with vanilla nova is janky, I hope it will get across the point.

Now, here is how it looks in laravel nova 4.
https://user-images.githubusercontent.com/33309407/176368360-0b3442cf-2f6a-4be4-93d1-3e60cec6cf57.mp4

Here is the filters response in laravel nova 4
![image](https://user-images.githubusercontent.com/33309407/176368458-083a7d22-cc2e-47a8-b3ec-0a031dd4672f.png)

Though this probably wont ever be an issue with vanilla nova, this does become a problem with custom packages that implement filters outside of the FilterMenu component. 
Example: 
https://github.com/outl1ne/nova-detached-filters

This functionality worked in version 3, which is why it seems like a bug since filters are built on top of vuex and should be using the same state.

### Detailed steps to reproduce the issue on a fresh Nova installation:
Create a filter and include it twice in resource filters() function return.

```php
    public function filters(Request $request)
    {
        return [
            ExampleFilter::make(),
            ExampleFilter::make(),
        ];
    }
 ```

**Comment by crynobone** [comment]
Majority of the changes were done to make it compatible with Vue 3, we wouldn't have changed those parts otherwise.

**Comment by KasparRosin** [comment]
@crynobone I'm guessing, since it was moved to a discussion, it's a not an issue/bug?
To me, it does not seem a cause of vue 3, since vuex is fully supports vue3.

**Reply by crynobone** [reply]
changes to computed to v-model cause this changes and this isn't a priority fix at the moment. If anyone want to send in a patch I be happy to try it out (against our dusk tests).

**Comment by KasparRosin** [comment]
Or can you point me in the direction of what is causing this in nova code? Maybe I can create a workaround for this specific package.
